src/main/java/edu/rosehulman/quota/Application.java

```diff
@@ -11,26 +11,26 @@ public static void main(String[] args) throws Exception {
     get("/health", (request, response) -> "OK");
 
     // Consume an AddUser call from SDK/partner
-    post("partner/:partnerId/product/:productId/user/:userId", new AddUserController());
+    post("partnerApi/:apiKey/product/:productId/user/:userId", new AddUserController());
```

Comment:
Why did you change this to `partnerApi`?

Reply:
Consistency with the addIUser that uses partnerApi.  All the time we refer to apiKey, that's what we use.